ops/k8s/deployments/grafana.yaml

```diff
@@ -58,12 +58,54 @@ spec:
         - mountPath: /var/lib/grafana/dashboards/bigquery-logs
           name: "grafana-config-bigquery-logs"
         env:
+        - name: GF_USERS_AUTO_ASSIGN_ORG_ROLE
+          value: "Editor"
         - name: GF_SERVER_HTTP_PORT
           value: "3000"
         - name: GF_SECURITY_ADMIN_PASSWORD
-          value: "helloworld"
+          valueFrom:
+            secretKeyRef:
+              name: google-clientid-secret
+              key: adminpassword
         - name: GF_INSTALL_PLUGINS
           value: "grafana-kubernetes-app,grafana-worldmap-panel,doitintl-bigquery-datasource"
+        - name: GF_AUTH_GOOGLE_ENABLED
+          value: "true"
+        - name: GF_AUTH_GOOGLE_ALLOW_SIGN_UP
+          value: "true"
```

Comment:
What does this mean? I think we only want to allow people with Broad/Verily accounts in. Does this line up with what you're thinking?

Reply:
It is only Broad Verily allowed, it sends out the oauth request and then stores the user locally

Reply:
```        - name: GF_AUTH_GOOGLE_ALLOWED_DOMAINS
          valueFrom:
            secretKeyRef:
              name: googleClientidSecret
              key: allowedDomains```

this is the specified domains

Reply:
You cannot camelCase the keys in the yaml